pkg/controller/command/didexchange/command_test.go

```diff
@@ -942,3 +950,41 @@ func generateKeyPair() (string, []byte) {
 
 	return base58.Encode(pubKey[:]), privKey
 }
+
+func newPeerDID(t *testing.T) *did.Doc {
+	pubKey, _, err := ed25519.GenerateKey(rand.Reader)
+	require.NoError(t, err)
+
+	key := did.PublicKey{
+		ID:         uuid.New().String(),
+		Type:       "Ed25519VerificationKey2018",
+		Controller: "did:example:123",
+		Value:      pubKey,
+	}
+	doc, err := peer.NewDoc(
+		[]did.PublicKey{key},
+		[]did.VerificationMethod{{
+			PublicKey:    key,
+			Relationship: 0,
+			Embedded:     true,
+			RelativeURL:  false,
+		}},
+		did.WithService([]did.Service{{
+			ID:              "didcomm",
+			Type:            "did-communication",
+			Priority:        0,
+			RecipientKeys:   []string{base58.Encode(pubKey)},
+			ServiceEndpoint: "http://example.com",
+		}}),
```

Comment:
default values can be removed e.g `Relationship, RelativeURL and Priority `

Reply:
@soluchok done